### ISSUE TITLE
new selectedDate on every render does not center

### ISSUE THREAD

**Issue by netpoetica** [clauderic/react-infinite-calendar#53]
When you re-render passing in a different selectedDate each time (where selected date is bound to state of container component), the component will not properly center on the selected date. The behavior can be seen by clicking around on different dates (usually the ones at the top or bottom of the calendar, because these need to be scrolled). This example is loosely based on Flexible Size example:

```
import React from 'react';
import InfiniteCalendar from 'react-infinite-calendar';
import 'react-infinite-calendar/styles.css';
import moment from 'moment';

export default class extends React.Component {
  constructor (props) {
    super(props);

    this.state = {
      selectedDate: moment()
    };
  }

  render () {
    let selectedDate = this.state.selectedDate;

    return (
      <InfiniteCalendar
        width={(window.innerWidth <= 650) ? window.innerWidth : 650}
        height={window.innerHeight - 250}
        rowHeight={70}
        selectedDate={selectedDate}
        afterSelect={(date) => {
          this.setState((prevState, currProps) => {
            return {
              selectedDate: date
            };
          });
        }}
      />
    );
  }
}
```

This is even more problematic when you have a short calendar, like so:

```
import React from 'react';
import InfiniteCalendar from 'react-infinite-calendar';
import 'react-infinite-calendar/styles.css';
import moment from 'moment';

export default class extends React.Component {
  constructor (props) {
    super(props);

    this.state = {
      selectedDate: moment()
    };
  }

  render () {
    let selectedDate = this.state.selectedDate;

    return (
      <InfiniteCalendar
        width={280}
        height={156}
        rowHeight={62}
        selectedDate={selectedDate}
        afterSelect={(date) => {
          this.setState((prevState, currProps) => {
            return {
              selectedDate: date
            };
          });
        }}
      />
    );
  }
}
```

You will notice that when you load the component, the selectedDate is not visible.

**Comment by ThomsCass** [comment]
for me the problem occurs when I give a value to the selectedDate property...

**Comment by clauderic** [comment]
Just shipped version `2.0.0`, and it should now properly re-centers to the new selected prop when it changes 👍

Check out #73 for more details
